hera_mc/cm_sysutils.py

```diff
@@ -313,16 +313,6 @@ def publish_summary(self, hlist=['default'], exact_match=False, hookup_cols='all
                       state='full', ports=True, revs=True,
                       sortby=sortby, filename=output_file, output_format='html')
 
-        from . import cm_transfer
-        if cm_transfer.check_if_main(self.session):  # pragma: no cover
-            import subprocess
-            location_on_web = 'hera.today:/var/www/html/hookup.html'
-            sc_command = 'scp {} {}'.format(output_file, location_on_web)
-            subprocess.call(sc_command, shell=True)
-            return 'OK'
-        else:
-            return 'Not on "main"'
-
```

Comment:
why are you taking this out?

Reply:
There is no longer hera.today, Matt will pull form the initial location.